### DISCUSSION TITLE
Failed to load resource: the server responded with a status of 500

### DISCUSSION THREAD

**Discussion by seowajik** [spencerwooo/onedrive-vercel-index#559] [Q&A] [answered]
I always have this, I'm using Microsoft Office 365 E5
![Screenshot_20220320-150523160 (1)](https://user-images.githubusercontent.com/101916347/159153868-a4a19f96-4572-4a11-9497-cd3f617a909f.jpg)

**Comment by spencerwooo** [comment]
Where did the `/:1` come from? Should be `/` only.

**Reply by seowajik** [reply]
I don't know where it came from, because I using default config `baseDirectory: '/',`

**Comment by seowajik** [comment, marked as answer]
I have solved this problem by using the default `api.config.js`... even though I have followed the tutorial by making `clientSecret` myself but suffer from the above errors continuously, For now I'm using the default `api.config.js` and looking for a way around this problem I've followed make `clientSecret` own but no results

**Reply by spencerwooo** [reply]
All right...good to hear that u found a workaround.
